content/projects/tdd/string-calculator-part-1/_index.md

```diff
@@ -139,26 +144,12 @@ In the first case, the default delimiter is: ";".
 
 In the second case, the default delimiter is: "4".
 
-**Note:** you are not expected to handle integer delimiters where the delimiter and the digit you are adding are the same. For example
-
-```
-add("//4\n14244")
-add("//88\n18882")
-```
-
-Strings like these should raise an error, `'ERROR: invalid input'`. When this isn't the case, your function should work as explained above;
-
-```
-add("//88\n18820882")
-// should return 23
-```
-
-**Python:** Raise an appropriate exception type. Using the base Exception isn't good practice. Here are [errors](https://www.tutorialsteacher.com/python/error-types-in-python) that can be raised.
+**Note:** you are not expected to handle integer delimiters where the delimiter and the digit you are adding are the same.
 
 #### 5. Modify the add function so that it can handle negative integers
 
-If a negative number is passed into the add function it should throw this exception:
-`"negatives not allowed"`
+If a negative number is passed into the add function it should throw this error/exception:
+`negatives not allowed <negative integers found>`
```

Comment:
Those angle brackets are confusing to look at.  Python and Js (and probably others) use {}  to indicate that there is a variable.  

How about `negatives not allowed {comma separated list of negative integers found}`